plugins/include-markdown/md-ast-to-mdx-ast.js

```diff
@@ -0,0 +1,13 @@
+const visit = require('unist-util-visit')
+const { isComment, getCommentContents } = require('@mdx-js/util')
+
+module.exports = (_options) => (tree) => {
```

Comment:
This is swiped directly from `@mdxjs/mdx`'s [`createMdxAstCompiler`](https://github.com/mdx-js/mdx/blob/510bae2580958598ae29047bf755b1a2ea26cf7e/packages/mdx/index.js#L18).

I considered the possibility of using `createMdxAstCompiler` rather than `remark-mdx` on its own - however, crucially, we do NOT want to transform our AST into a MDXAST, we ONLY want to transform custom component nodes (ie HTML that is really JSX) into JSX nodes. So it felt duplicative but necessary to copypasta this utility in to meet our needs.

Reply:
It would be helpful perhaps to include a note of this nature as a comment in this file - this way anyone coming back to this in the future would have a little context on what it is and why its done this way 😀

Reply:
Great callout, ty 💖 added a comment ✅